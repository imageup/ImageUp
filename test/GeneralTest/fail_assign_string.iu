function main() -> int
{
	string: s;
	s = 1.9;
	return 0;
}