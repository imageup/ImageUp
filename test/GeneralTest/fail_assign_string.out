Fatal error: exception Failure("illegal assignment string = float in s = 1.9")